config/rbac/role.yaml

```diff
@@ -182,6 +182,24 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
we need a separate role for kruise-daemon , it is important that kruise-daemon use the minimum privilidge